qark/plugins/manifest/exported_tags.py

```diff
@@ -194,18 +193,21 @@ def _check_manifest_issues(self, possibly_vulnerable_tag, tag, file_object):
 
                 if protected:
                     # intent filter has protected actions
-                    issues.add(Vulnerability(category="Manifest", issue_name="Protected Exported Tags",
-                                             severity=Severity.INFO,
-                                             description=EXPORTED_IN_PROTECTED.format(tag=tag, tag_name=tag_name),
-                                             file_object=file_object))
+                    self.issues.add(Vulnerability(category="Manifest", issue_name="Protected Exported Tags",
+                                                  severity=Severity.INFO,
+                                                  description=EXPORTED_IN_PROTECTED.format(tag=tag, tag_name=tag_name),
+                                                  file_object=file_object))
                 elif has_permission and self.min_sdk < 20:
-                    issues.add(Vulnerability(category="Manifest", issue_name="Exported Tag With Permission",
-                                             severity=Severity.INFO,
-                                             description=EXPORTED_AND_PERMISSION_TAG.format(tag=tag, tag_name=tag_name),
-                                             file_object=file_object))
+                    self.issues.add(Vulnerability(category="Manifest", issue_name="Exported Tag With Permission",
+                                                  severity=Severity.INFO,
+                                                  description=EXPORTED_AND_PERMISSION_TAG.format(tag=tag,
+                                                                                                 tag_name=tag_name),
+                                                  file_object=file_object))
                 else:
-                    issues.add(Vulnerability(category="Manifest", issue_name="Exported Tags",
-                                             severity=Severity.WARNING,
-                                             description=EXPORTED.format(tag=tag, tag_name=tag_name),
-                                             file_object=file_object))
-        return issues
+                    self.issues.add(Vulnerability(category="Manifest", issue_name="Exported Tags",
+                                                  severity=Severity.WARNING,
+                                                  description=EXPORTED.format(tag=tag, tag_name=tag_name),
+                                                  file_object=file_object))
+
+
+plugin = ExportedTags()
```

Comment:
Don't think that we should create a module wide instance of the plugin. Should let users instantiate for themselves

Reply:
PluginBase lets us import things as a module, but each plugin will have a different class name. That means that to instantiate it, we need to know the class name. We won't know how to instantiate them with different names, so each plugin module creates an instance of itself so that we can just do 

```python
plugin = plugin_source.load_plugin(plugin_name).plugin
```

To know what plugin to run.

Reply:
Makes sense